src/bci_build/package/python.py

```diff
@@ -45,14 +45,20 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
     py3_ver_replacement = f"%%py{py3_ver_nodots}_ver%%"
     pip3 = f"{py3}-pip"
     pip3_replacement = "%%pip_ver%%"
-    has_pipx = False
-    has_wheel = True if is_system_py or os_version == OsVersion.TUMBLEWEED else False
-    # Tumbleweed rocks
-    if os_version == OsVersion.TUMBLEWEED:
-        has_pipx = True
-    # Enabled only for Python 3.11 on SLE15 (jsc#PED-5573)
-    if os_version not in (OsVersion.BASALT, OsVersion.TUMBLEWEED) and py3_ver == "3.11":
+    has_pipx: bool = False
+    has_wheel: bool = is_system_py
+
+    if os_version.is_tumbleweed:
+        # Tumbleweed rocks
         has_pipx = True
+        has_wheel = True
+    elif os_version.is_sle15:
+        if py3_ver != "3.6":
+            # Enabled only for Python 3.11+ on SLE15 (jsc#PED-5573)
+            has_pipx = True
```

Comment:
```suggestion
        # Enabled only for Python 3.11+ on SLE15 (jsc#PED-5573)
        has_pipx = is_system_py
```

Reply:
no, that would be the inverse of the condition..